src/server/user-controller.js

```diff
@@ -310,6 +311,18 @@ class UserController {
     res.status(201).send({ user: updatedUser });
   }
 
+  async handlePostUserNotificationSettings(req, res) {
+    const userId = req.user._id;
+    const { emailNotificationFrequency } = req.body;
+    const updatedUser = await this.userService.updateUserNotificationSettings({ userId, emailNotificationFrequency });
+
+    if (!updatedUser) {
+      throw new NotFound();
+    }
+
+    res.status(201).send({ user: updatedUser });
```

Comment:
For consistency and to avoid possible future issues (though right now I don't think there are issues), let's do a `this.clientDataMappingService.mapWebsiteUser(updatedUser)` mapping, like we do in all other cases where we return the data of the logged in user (equivalently for a viewed user we use `mapWebsitePublicUser`)

Reply:
Very good point, thanks! Changed it (here and also in the case for the profile update).

Reply:
Wunderbar 👍